js_modules/dagit/packages/core/src/partitions/PartitionGraph.tsx

```diff
@@ -5,25 +5,23 @@ import styled from 'styled-components/macro';
 
 import {colorHash} from '../app/Util';
 
-import {PartitionGraphFragment} from './types/PartitionGraphFragment';
-
 type PointValue = number | null | undefined;
 type Point = {x: string; y: PointValue};
 
 interface PartitionGraphProps {
-  runsByPartitionName: {[name: string]: PartitionGraphFragment[]};
-  getPipelineDataForRun: (run: PartitionGraphFragment) => PointValue;
-  getStepDataForRun: (run: PartitionGraphFragment) => {[key: string]: PointValue[]};
+  partitionNames: string[];
+  pipelineDataByPartition?: {[partitionName: string]: PointValue};
+  stepDataByPartition?: {[partitionName: string]: {[key: string]: PointValue[]}};
```

Comment:
Kinda whacky for this to still mention pipeline, I wonder if we could rename this to  jobDataByPartition?